modules/git/git.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // RequiredVersion is the minimum Git version required
-const RequiredVersion = "2.0.0"
+const RequiredVersion = "2.24.0"
```

Comment:
Oh, no. It's a big break.

Reply:
debian 10 ships git 2.20, every other big distro i checked ships at least 2.25.
https://pkgs.org/search/?q=%2Fusr%2Fbin%2Fgit

Reply:
I could agree to make Gitea require Git 2.24 if we can bear user's complaining (I can bear)

I know many users are still running some OS like Ubuntu 16 or 18.